lib/src/base-styles/_nx-lists.scss

```diff
@@ -20,101 +19,94 @@
   }
 }
 
-.nx-list__title {
-  color: $nx-text-color;
-  margin: 0;
-}
-
 .nx-list__item {
-  @include semi-bold();
-  @include nx-truncate-ellipsis();
+  @include container-horizontal;
   @include container-vertical;
 
-  border-top: 1px solid transparent;
-  border-bottom: $nx-border-row;
+  border-top: 1px solid #e9e9e9;
   box-sizing: border-box;
-  color: $nx-text-color;
+  display: grid;
+  grid-template-columns: 1fr auto;
+  grid-template-rows: auto auto;
+  grid-template-areas: 
+    "list-text actions"
+    "list-sub-text actions";
   margin: 0;
-  padding: $nx-spacing-sm $nx-spacing-xs;
+  padding: $nx-spacing-md;
 
   // we use first of type because we only want to modify the first list item,
   // not alerts, errors, or empty states
```

Comment:
Does this comment (still) make sense?

Reply:
> Does this comment (still) make sense?

Not anymore: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/e5aa4fb76422eebda7c77948216f8d1cee0e3d2b

Reply:
> Not anymore: e5aa4fb

Shouldn't it also be using `:first-child` instead of `:first-of-type`?

Reply:
> > Not anymore: [e5aa4fb](https://github.com/sonatype/sonatype-react-shared-components/commit/e5aa4fb76422eebda7c77948216f8d1cee0e3d2b)
> 
> Shouldn't it also be using `:first-child` instead of `:first-of-type`?

Sure: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/0217e1e92483f0b75dfc2a9f45c40d1861937de6